### DISCUSSION TITLE
Thumbnails and pictures don't load, but videos do.

### DISCUSSION THREAD

**Discussion by OscarArmend** [photoprism/photoprism#3075] [Q&A]
Ok, I've been reading and trying to find solutions and I came back empty. If there is a thread with this issue, please let me know, but I couldn't find it. 
I have over 100 thousand pictures, I have them index, but no thumbnails show up, not for picture and not for videos. When I click on a video, it will play, almost instantly, but when I click a picture, it will never load. 
I don't know what information you need to help me, so please let me know what you need to see, and I will provide. 
I have Unraid ver 6.1.1.5setup on a dedicated computer (Intel® Core™ i5-4590 CPU @ 3.30GHz) I have a healthy parity disk, and 2 1 TB hard drives. I do have a Cache driver which is not being used with PhotoPrism. 
In the PhotoPrisim logs, I do see cache errors:

These are not all of them, but they all are the same, just with a different time stamp... 
```
time="2023-01-08T12:25:08-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:25:08-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:25:08-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:27:02-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:27:02-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:27:02-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:27:44-08:00" level=error msg="thumb: not cached"
time="2023-01-08T12:27:44-08:00" level=error msg="thumb: not cached"
```

My cache is set to save here: /mnt/cache/appdata/photoprism
My pictures are saved here: /mnt/user/Pictures
My thumbnail size is the default: 2048
On-demand Size Limit: 7680
Converted Image Size Limit: 7680
Thumbnail Quality: 90
Resample Filter: lanczos

I appreciate any input.

**Comment by lastzero** [comment]
Seems like the thumbs got lost. Is this a temporary folder that gets erased after a restart? Reindexing should fix this, but only as long as files don't get lost again. Also, you can enable dynamic / on demand rendering of thumbnails.

**Reply by OscarArmend** [reply]
I reindexed just a few days ago, and it didn't help. This is not meant to be a temporary folder, and I don't think is getting lost. Should I set the thumbnail folder to be the same as the photo folder?

**Reply by lastzero** [reply]
With indexed you mean a full rescan? You also checked filesystem permissions and went through our Troubleshooting Checklists we provide for this purpose?

**Reply by OscarArmend** [reply]
Yes, full rescan. No sir, I did not go through the troubleshooting checklist, I guess I did not know there was one. Where could I find it?

**Reply by lastzero** [reply]
https://docs.photoprism.app/getting-started/troubleshooting/#broken-thumbnails
